juju/juju
# chore: use instance.Id as return type for instance methods

Replaces the string that was naively used for the InstanceID() and InstanceIDAndName() methods from machine domain.





## Checklist




## QA steps

Only type change on method signatures, so this should compile and unit tests pass.


## Links


**Jira card:** JUJU-6901
